commands/qx.go

```diff
@@ -179,7 +179,7 @@ var wifToPriCmd = &cobra.Command{
 	Use:   "wiftopri {string, wif}",
 	Short: "WIF key to private key",
 	Example: `
-		wiftopri  5HueCGU8rMjxEXxiPuD5BDku4MkFqeZyd4dZ1jvhTVqvbTLvyTJ
+		wiftopri  PxBefLecRtTYPoxUUwAbq8m7xGmzDK8gQ71N9qKyhp2j5yN42Rpzc
```

Comment:
I doubt `wiftopri` can handle this without appointing wallet import format since we have two formats now. 
- option1, `wiftopri` means to support **WIF** format only , if true, the example should not show the `Pxxxxx` privatekey.
- option2. add the support of two import formats by using the `--format` argument.

Reply:
ok